lib/metalsmith.js

```diff
@@ -52,7 +52,6 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
   .use(concat({
     files: [
       'javascripts/vendor/jquery.js',
-      'javascripts/vendor/modernizr.js',
```

Comment:
How is Modernizr being copied into the build directory now?

Reply:
the assets are being copied by default if they're not in the `pipe` of another plugin. I should probably document that somewhere.

Reply:
Gotcha.